cherryontech/do-five-things
# feat: Add Today and Progress pages

## Context
Devs setting up pages :)

## Work done
- Added routes and mock content for progress and today pages
- Added nav routes for the new pages

## Manual testing instructions
1. serve app
2. Navigate to /settings, /today, and /progress
- Expect to see mock content on each page with the name of the page
